docs/docs/configuration/settings.md

```diff
@@ -72,14 +72,22 @@ The configuration file, `configuration.json` is in the Oni2 directory, whose loc
 
 - `editor.zenMode.hideTabs` __(_bool_ default: `true`)__ - When `true`, the Onivim will hide the buffer tabs from the user whilst in zen mode. Zen mode can be toggled in the command pallette, or automatically enabled with the `editor.zenMode.singleFile` configuration option.
 
-- `workbench.colorTheme` __(_string)_ default:`"One Dark Pro"`)__ - Color theme to use.
```

Comment:
The closing parentheses around `string` is a typo.

Reply:
Ah, great catch - thanks for fixing this!